src/Illuminate/Database/Query/Builder.php

```diff
@@ -2322,23 +2322,6 @@ public function useWritePdo()
     }
 
     /**
-     * Use a different connection for the query.
-     *
-     * @param  \Illuminate\Database\ConnectionInterface  $connection
-     * @param  \Illuminate\Database\Query\Grammars\Grammar  $grammar
-     * @param  \Illuminate\Database\Query\Processors\Processor  $processor
-     * @return void
-     */
-    public function useConnection(ConnectionInterface $connection,
```

Comment:
I removed this helper method since it's not needed any more. It was added here https://github.com/laravel/framework/pull/13784 and hasn't been tagged yet.